src/load.rs

```diff
@@ -76,10 +65,19 @@ impl Loader {
         loader
     }
 
+    /// Convert a path string to a FileId.  For performance reasons
+    /// this requires an owned 'path' param.
+    fn path(&mut self, mut path: String) -> FileId {
```

Comment:
It's too bad, I went to all the effort to make this area reuse a single String buffer, but I understand why it must be done.  RIP